types/google-apps-script/google-apps-script-events.d.ts

```diff
@@ -20,6 +21,29 @@ declare namespace GoogleAppsScript {
       user: Base.User,
     }
 
+    interface AppsScriptHttpRequestEvent {
+        parameter: object,
+        contextPath: string,
+        contentLength: number
+        queryString: string,
+        parameters: object,
+      }
+
+    /**
+     * TODO: What about other properties? Object.keys(e.postData) contains:
+     * copyBlob, getAllBlobs, getAs, getBlob, getBytes, getContentType,
+     * getDataAsString, getDocId, getImageUrl, getName, getPrimitiveByteArray,
+     * isGoogleType, requireContentType, requireData, requireName, setBytes,
+     * setContentType, setContentTypeFromExtension, setDataFromString,
+     * setDataFromString, setName, toString
+     */
+    interface AppsScriptHttpRequestEventPostData {
+        length: number,
+        type: string,
+        contents: string,
+        name: string, // Always the value "postData"
+      }
```

Comment:
Bad formatting

Reply:
I returned the formatting as it is in this directory.